yt_dlp/extractor/fancode.py

```diff
@@ -38,16 +39,54 @@ class FancodeVodIE(InfoExtractor):
         'only_matching': True,
     }]
 
+    _ACCESS_TOKEN = None
+    _REFRESH_TOKEN = None
+    _NETRC_MACHINE = 'fancode'
+
+    headers = {
+        'content-type': 'application/json',
+        'origin': 'https://fancode.com',
+        'referer': 'https://fancode.com',
+    }
+
+    def _login(self):
+        # Access tokens are shortlived, so get them using the refresh token.
```

Comment:
If user provides cookies, cant refresh token be extracted from there?

Reply:
I didn't find any way to do that. They only seem to set when a user first logs in.